ground/src/main/java/com/google/android/ground/persistence/remote/firebase/FirestoreDataStore.kt

```diff
@@ -140,6 +142,12 @@ internal constructor(
       .subscribeOn(schedulers.io())
   }
 
+  override fun subscribeToSurveyUpdates(surveyId: String): Completable =
+    RxTask.toCompletable {
+      Timber.d("Subscribing to FCM topic $surveyId")
+      Firebase.messaging.subscribeToTopic(surveyId)
```

Comment:
What happens if we switch the survey? Does the old subscription get expired or will it still be active?

Reply:
For now it stays active. To fix this, we unsubscribe when the user turns off "available offline" for synced surveys as part of #450.